_webi/bootstrap.sh

```diff
@@ -42,8 +42,11 @@ set -u
 
 __webi_main() {
 
-    export WEBI_TIMESTAMP="\$(date +%F_%H-%M-%S)"
-    export _webi_tmp="\${_webi_tmp:-\$(mktemp -d -t webi-\$WEBI_TIMESTAMP.XXXXXXXX)}"
+    my_date="\$(date +%F_%H-%M-%S)"
+    export WEBI_TIMESTAMP="\${my_date}"
+    export _webi_tmp="\${_webi_tmp:-\$(
+        mktemp -d -t "webi-\$WEBI_TIMESTAMP.XXXXXXXX"
```

Comment:
It seems that just putting this on its own line got rid of a linter warning.

That may mean that it's a false negative and the command should still be lifted later.